src/BadWord.js

```diff
@@ -0,0 +1,264 @@
+const Discord = require('discord.js');
+
+/**
+ * Database
+ * @type {Database}
+ */
+let database;
+
+/**
+ * Config cache time (ms)
+ * @type {Number}
+ */
+const cacheDuration = 10*60*1000;
+
+/**
+ * channel wide bad words
+ * @type {module:"discord.js".Collection}
+ */
+const channels = new Discord.Collection();
+
+/**
+ * guild wide bad words
+ * @type {module:"discord.js".Collection}
+ */
+const guilds = new Discord.Collection();
+
+/**
+ * Class representing a bad word
+ */
+class BadWord {
```

Comment:
It probably makes more sense to create a parent class for `AutoResponse` and `BadWord` to share similar functions like `matches()`.

Reply:
matches() and triggerTypes are the only matching properties I found